lebedigital/demonstrator_scripts/beam_design.py

```diff
@@ -200,11 +208,20 @@ def check_design(span:float,
         in the bottom of the section. It is negative if  design is not satisfied and positive if design is satisfied.
         Optimal will be close to zero.
     """
-    max_moment, max_shear_force = max_bending_moment_and_shear_force(span, point_load, distributed_load)
-    design = beam_section_design(width, height, max_moment, max_shear_force, compr_str_concrete, yield_str_steel, steel_dia, cover)
-    specified_area = (math.pi * steel_dia ** 2 / 4) * n_bottom
+    max_moment, max_shear_force = max_bending_moment_and_shear_force(span,
+                                                                     point_load,
+                                                                     distributed_load)
+
+    design = beam_section_design(width,
+                                 height,
+                                 max_moment,
+                                 max_shear_force,
+                                 compr_str_concrete,
+                                 yield_str_steel,
+                                 steel_dia ,
+                                 cover)
+
+    specified_area = (math.pi * (steel_dia) ** 2 / 4) * n_bottom
```

Comment:
if area has a unit, we could remove that in the key for the dict that is returned

Reply:
Correct! Will do so :)